unstructured/ingest/utils/table.py

```diff
@@ -0,0 +1,24 @@
+import typing as t
+
+import pandas as pd
+
+from unstructured.staging.base import flatten_dict, get_default_dtypes
+
+
+def convert_to_df(
```

Comment:
```suggestion
def convert_to_dataframe(
```
for readability? ..then I know at a glance exactly what this is